src/kete/horizons.py

```diff
@@ -318,21 +318,14 @@ def _sample(self, n_samples):
         "peri_time",
     ]
 
-    labels = self.json["orbit"]["covariance"]["labels"]
```

Comment:
This was poorly implemented before, this along with the Time scaling, I did a little cleanup.